sw/device/lib/base/BUILD

```diff
@@ -120,33 +120,12 @@ cc_library(
     ],
 )
 
-cc_library(
```

Comment:
Can you comment on why this is being removed?  I don't necessarily object, but:

The original motivation here was to prevent the compiler provided 64-bit division functions from landing in our code. Things like `__udivdi3` are rather large by comparison to how much ROM space we have available.

Reply:
If you do a 64-bit division you'll get a linker error anyway: `undefined symbol: __udivdi3`. But I'll see if I can avoid adding that commit.

Reply:
@cfrantz I have checked that these `__udivdi3` etc functions are hard-coded into the compiler and won't be removed during LTO, even if they aren't referenced. So we can't keep the current solution. In any case, I don't think we need it anyway, as we still get a linker error if we unintentionally do a 64-bit division, etc.